code/modules/reagents/reagent_containers/pill_vr.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/weapon/reagent_containers/pill/nutriment
+	name = "Nutriment pill"
+	desc = "Used to feed people on the field. Contains 30 units of Nutriment."
+	icon_state = "pill6"
+
+/obj/item/weapon/reagent_containers/pill/iron/New()
+	..()
+	reagents.add_reagent("nutriment", 30)
```

Comment:
A change to stuff that regularly spawns on the normal map in a PR titled "ERT equipment redo" and isn't listed in the list of changes? Gettin' tired of yer shit Ascian.

Reply:
When I copy-pasta'd I forgot to change iron to nutriment.